src/apikey/lib/redis.js

```diff
@@ -11,29 +12,40 @@ if (!fs.pathExists('../apikey.json')) {
   logger.warn('No API key are set');
 }
 
-if (process.env.NODE_ENV === 'production') {
-  apiKeys = require('../apikey.json');
-} else {
-  apiKeys = require('../apikey-dev.json');
-}
-
 const redisClient = redis.createClient({
   host: config.get('redis.host'),
   port: config.get('redis.port'),
   password: config.get('redis.password'),
 });
 
+redisClient.get = util.promisify(redisClient.get);
+redisClient.del = util.promisify(redisClient.del);
 redisClient.ping = util.promisify(redisClient.ping);
+redisClient.set = util.promisify(redisClient.set);
+redisClient.keys = util.promisify(redisClient.keys);
+redisClient.flushall = util.promisify(redisClient.flushall);
 
 redisClient.on('error', (err) => {
   logger.error(`Error in redis ${err}`);
 });
 
-const apiKeysJSON = Object.keys(apiKeys);
+const load = async () => {
+  if (process.env.NODE_ENV === 'production') {
+    apiKeys = await fs.readFile(path.resolve(__dirname, '..', 'apikey.json'));
+  } else {
+    apiKeys = await fs.readFile(path.resolve(__dirname, '..', 'apikey-dev.json'));
+  }
```

Comment:
```suggestion
  const filename = process.env.NODE_ENV === 'production' ? 'apikey.json' : 'apikey-dev.json';
  apiKeys = await fs.readFile(path.resolve(__dirname, '..', filename), 'utf8');
```
Just another way to do the same thing.

Reply:
ok i follow that